packages/design-system/src/components/tree/tree.tsx

```diff
@@ -265,7 +265,11 @@ export const Tree = <Data extends { id: string }>({
       onScroll={dropHandlers.handleScroll}
     >
       <Box
+        // styled should support passing { current: T | undefined } to ref
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore React.MutableRefObject cannot be passed to React.RefObject
```

Comment:
that's weird

Reply:
The error is correct, ref should be defined like this: `const rootRef = useRef<HTMLDivElement | null>(null);`

Reply:
that's normal yeah

Reply:
@TrySound please ask for help @Andarist in case of typescript issues

Reply:
fixed it https://github.com/webstudio-is/webstudio-designer/commit/1a66acb79c8c560b3a54b55512c2779991a68eff